qa/rpc-tests/wallet_persistence.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python2
+# Copyright (c) 2018 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_true, start_nodes, stop_nodes, \
+    wait_bitcoinds
+
+class WalletPersistenceTest (BitcoinTestFramework):
```

Comment:
It would be good to update this test to make sure that we can still spend a sapling note after having restarted.

Reply:
I thought there was a test that does this, but I need to confirm. We should at least put a note in one of the test that assumes this test ensures a disk persists is valid.

Reply:
Pushed a commit to update the test.